### ISSUE TITLE
J and K keys no longer navigate between tweets in Twitter app (using NVDA 2021.1 beta 1 & 2)

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#12527]
### Steps to reproduce:

1. Open the Twitter app
2. Turn off NVDA's single letter navigation (NVDA+shift+spacebar
3. Press J or K to navigate between tweets

### Actual behavior:

Using NVDA 2020.4, NVDA reads the next tweet as focus moves.
Using NVDA 2021.1 Beta 1 or Beta 2, the focus moves but NVDA does not read anything.  Reading manually (eg reading current line) the focus is still on the previous tweet.

When running the 2021.1 betas, the following log fragment was generated:

```
IO - inputCore.InputManager.executeGesture (09:46:43.732) - winInputHook (21940):
Input: kb(desktop):k
DEBUGWARNING - NVDAObjects.UIA.UIA._prefetchUIACacheForPropertyIDs (09:46:43.782) - MainThread (1284):
IUIAutomationElement.buildUpdatedCache failed given IDs of {30019, 30086, 30022, 30025, 30155, 30036, 30070, 30103, 30008, 30009, 30010, 30046, 30079}
DEBUGWARNING - eventHandler.executeEvent (09:46:43.790) - MainThread (1284):
error executing event: typedCharacter on <NVDAObjects.UIA.spartanEdge.EdgeNode object at 0x0387E050> with extra args of {'ch': 'k'}
Traceback (most recent call last):
  File "eventHandler.pyc", line 246, in executeEvent
  File "eventHandler.pyc", line 96, in __init__
  File "eventHandler.pyc", line 105, in next
  File "NVDAObjects\__init__.pyc", line 1082, in event_typedCharacter
  File "speech\speech.pyc", line 1027, in speakTypedCharacters
  File "api.pyc", line 272, in isTypingProtected
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\__init__.pyc", line 937, in _get_isProtected
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\web.pyc", line 364, in _get_states
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 1425, in _get_states
  File "NVDAObjects\UIA\__init__.pyc", line 839, in _getUIACacheablePropertyValue
  File "comtypesMonkeyPatches.pyc", line 27, in __call__
_ctypes.COMError: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.UIA.UIA._prefetchUIACacheForPropertyIDs (09:46:43.813) - MainThread (1284):
IUIAutomationElement.buildUpdatedCache failed given IDs of {30019, 30086, 30022, 30025, 30155, 30036, 30070, 30103, 30008, 30009, 30010, 30046, 30079}
DEBUGWARNING - eventHandler.executeEvent (09:46:43.814) - MainThread (1284):
error executing event: gainFocus on <NVDAObjects.UIA.spartanEdge.EdgeNode object at 0x034DDE70> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 246, in executeEvent
  File "eventHandler.pyc", line 96, in __init__
  File "eventHandler.pyc", line 105, in next
  File "browseMode.pyc", line 1569, in event_gainFocus
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\web.pyc", line 364, in _get_states
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 1425, in _get_states
  File "NVDAObjects\UIA\__init__.pyc", line 839, in _getUIACacheablePropertyValue
  File "comtypesMonkeyPatches.pyc", line 27, in __call__
_ctypes.COMError: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
```

Reading Twitter in Edge or Firefox, the J and K keys work as expected in this situation.

### Expected behavior:

NVDA should move focus to the next (or previous) tweet and read it.

#### NVDA version:

NVDA 2021.1 Beta 2 installed
NVDA 2020.4 portable

#### Windows version:

Windows 10 (64-bit) Version: 2009, Build: 19042 

#### Other information about your system:

Twitter app (no version information available?)

**Comment by Qchristensen** [comment]
Not sure why the text got all jumbled up there.  Here's the same log fragment as text:

[twitterlog.txt](https://github.com/nvaccess/nvda/files/6612613/twitterlog.txt)

**Comment by Surveyor123** [comment]
Below adding a piece of log generated through navigating among the player controls of windows 10 netflix app. All addons are disabled.

error executing event: gainFocus on <NVDAObjects.UIA.spartanEdge.EdgeNode object at 0x03F9E4B0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 246, in executeEvent
  File "eventHandler.pyc", line 96, in __init__
  File "eventHandler.pyc", line 105, in next
  File "browseMode.pyc", line 1589, in event_gainFocus
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\web.pyc", line 364, in _get_states
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 1421, in _get_states
  File "NVDAObjects\UIA\__init__.pyc", line 838, in _getUIACacheablePropertyValue
  File "comtypesMonkeyPatches.pyc", line 27, in __call__
_ctypes.COMError: (-2147220991, 'Bir olay abonelerden birini çağıramadı', (None, None, None, 0, None))
IO - inputCore.InputManager.executeGesture (09:30:43.947) - winInputHook (12080):
Input: kb(laptop):shift+tab
DEBUGWARNING - NVDAObjects.UIA.UIA._prefetchUIACacheForPropertyIDs (09:30:44.017) - MainThread (10908):
IUIAutomationElement.buildUpdatedCache failed given IDs of {30019, 30086, 30022, 30025, 30155, 30036, 30070, 30103, 30008, 30009, 30010, 30046, 30079}
DEBUGWARNING - eventHandler.executeEvent (09:30:44.019) - MainThread (10908):
error executing event: gainFocus on <NVDAObjects.UIA.spartanEdge.EdgeNode object at 0x093607B0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 246, in executeEvent
  File "eventHandler.pyc", line 96, in __init__
  File "eventHandler.pyc", line 105, in next
  File "browseMode.pyc", line 1589, in event_gainFocus
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\web.pyc", line 364, in _get_states
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 1421, in _get_states
  File "NVDAObjects\UIA\__init__.pyc", line 838, in _getUIACacheablePropertyValue
  File "comtypesMonkeyPatches.pyc", line 27, in __call__
_ctypes.COMError: (-2147220991, 'Bir olay abonelerden birini çağıramadı', (None, None, None, 0, None))
IO - inputCore.InputManager.executeGesture (09:30:45.351) - winInputHook (12080):
Input: kb(laptop):shift+tab
DEBUGWARNING - NVDAObjects.UIA.UIA._prefetchUIACacheForPropertyIDs (09:30:45.397) - MainThread (10908):
IUIAutomationElement.buildUpdatedCache failed given IDs of {30019, 30086, 30022, 30025, 30155, 30036, 30070, 30103, 30008, 30009, 30010, 30046, 30079}
DEBUGWARNING - eventHandler.executeEvent (09:30:45.407) - MainThread (10908):
error executing event: gainFocus on <NVDAObjects.UIA.spartanEdge.EdgeNode object at 0x03F8C3B0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 246, in executeEvent
  File "eventHandler.pyc", line 96, in __init__
  File "eventHandler.pyc", line 105, in next
  File "browseMode.pyc", line 1589, in event_gainFocus
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\web.pyc", line 364, in _get_states
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 146, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 1421, in _get_states
  File "NVDAObjects\UIA\__init__.pyc", line 838, in _getUIACacheablePropertyValue
  File "comtypesMonkeyPatches.pyc", line 27, in __call__
_ctypes.COMError: (-2147220991, 'Bir olay abonelerden birini çağıramadı', (None, None, None, 0, None))
IO - inputCore.InputManager.executeGesture (09:30:46.547) - winInputHook (12080):
